minimal/docker-compose.yml

```diff
@@ -45,17 +43,20 @@ services:
       - ./media:/code/media
       - ./staticfiles:/code/staticfiles
     ports:
-      - "0.0.0.0:80:80"
+      - "80:80"
+      - "443:443"
 
   {{ project_name }}_certbot:
     restart: always
     build: ./compose/certbot/production
+    volumes:
+      - {{ project_name }}_https_certs:/etc/letsencrypt
+      - ./staticfiles:/code/staticfiles
     depends_on:
       - {{ project_name }}_nginx
-    volumes_from:
-      - {{ project_name }}_nginx
+      - {{ project_name }}_web
     env_file: production.env
 
-
 volumes:
   {{ project_name }}_data:
+  {{ project_name }}_https_certs:
```

Comment:
Since this volume will be created by the startup script, we need to state that it's external, so change to:

```
volumes:
  {{ project_name  }}_https_certs:
    external: true
```